### ISSUE TITLE
Completion expiry should use timestamp instead of stored

### ISSUE THREAD

**Issue by garemoko** [davidpesce/moodle-mod_tincanlaunch#125]
Comparing the expiry to a timestamp is better than to the stored date because what really matters is when the learner did the thing, not when it was stored in the LRS. 

Modify tincanlaunch_get_completion_state in lib.php to not just check if statements are returned but also cycle through the statements and make sure that at least one has a timestamp greater than the expiry date. 

Currently the relevant code looks like this:
```
        // If the statement exists, return true else return false.
        if (!empty($statementquery->content) && $statementquery->success) {
            $result = true;
        } else {
            $result = false;
        }
```

Note: setting the result variable to true/false and then returning the variable is unneeded. We can just return true or false.

**Comment by davidpesce** [comment]
Fixed in #198

**Comment by davidpesce** [comment]
@garemoko - I thought I had fixed this an in testing realized that the tincanlaunch_get_statements is returning a new LRSResponse object. That object has an array of protected "content" and not accessible. 

I stumbled on your answer here: https://stackoverflow.com/questions/35693090/fetching-statements-from-learning-locker-lrs-using-tincan-api and went back to look at the tincanlaunch_get_statements method and it is requesting the getStatement on each of the statements.

I tried removing the returned LRSResponse object and simply passing the allthestatements array, but those are still protected as well. 

Any ideas?
